### PR TITLE
Topic 01 excercise 06

### PR DESCRIPTION
**Pull request by vvale93**

### CODE REVIEW

File: Topic-01-Excercise-04/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Github Search</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header><h1>Another awesome page</h1></header>
+        <section class="section">
+            <article>
+                <form>
+                    <label for="inputText">Search: </label><input type="text" value=""  id="inputText" placeholder="What are you looking for?">
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/index.html]: Dont render several tags in the same line please

File: Topic-01-Excercise-04/script.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                xhr.send();
+            });
+        };
+        request(config)
+            .then(data => {
+                let response = JSON.parse(data); 
+                let items = response.items;
+                for(let i = 0; i<items.length; i++)
+                {
+                document.getElementById("results").innerHTML += "<li><a href=\"https://github.com/"+items[i].full_name+"\">"+items[i].name+"</a></li>";
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/script.js]: Use Template literals

File: Topic-01-Excercise-06/script.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                let tag = document.createElement('td');
+                tag.setAttribute('class', 'table element');
+                let content = document.createTextNode(matrix[rows][columns]);
+                tag.appendChild(content);
+                eachRow.appendChild(tag);
+            }
+        }
+        ref.appendChild(eachRow);
+    }
+}(matrix));
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-06/script.js]: Why are you using IIFE ? please explain

File: Topic-01-Excercise-06/index.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <section>
+        <article>
+            <table id="table">
+
+            </table>
+        </article>
+    </section>
+    <script src="script.js"></script>
+</body>
+</html>
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-06/index.html]: New line after this please

File: Topic-01-Excercise-06/script.js (hunk 3 above)
Comment (DemianMoschin-GLB) [reply Topic-01-Excercise-06/script.js]: Add empty line

**Comment by vvale93** [comment]
done

File: Topic-01-Excercise-04/index.html (hunk 1 above)
Comment (vvale93) [reply Topic-01-Excercise-04/index.html]: done

File: Topic-01-Excercise-04/script.js (hunk 2 above)
Comment (vvale93) [reply Topic-01-Excercise-04/script.js]: done

File: Topic-01-Excercise-06/index.html (hunk 4 above)
Comment (vvale93) [reply Topic-01-Excercise-06/index.html]: done

File: Topic-01-Excercise-06/script.js (hunk 3 above)
Comment (vvale93) [reply Topic-01-Excercise-06/script.js]: i changed the iife function and added empty line

File: Topic-01-Excercise-04/script.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+function onValueChange()
+{
+    let search = inputText.value;
+    if (search) 
+    {
+        let config = {
+            method: 'GET',
+            url: 'https://api.github.com/search/repositories?q=' + search,
```
Comment (DemianMoschin-GLB) [inline Topic-01-Excercise-04/script.js]: Template literals...

**Pull request merged**